modules/oe_multilingual_url_suffix/src/Form/NegotiationUrlSuffixForm.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Drupal\oe_multilingual_url_suffix\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation\LanguageNegotiationUrlSuffix;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Configure the URL suffixes language negotiation method for this site.
+ *
+ * @package Drupal\oe_multilingual_url_suffix\Form
+ */
+class NegotiationUrlSuffixForm extends ConfigFormBase {
+
+  /**
+   * The language manager.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
+  /**
+   * Constructs a new NegotiationUrlSuffixForm object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The factory for configuration objects.
+   * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
+   *   The language manager.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, LanguageManagerInterface $language_manager) {
+    parent::__construct($config_factory);
+    $this->languageManager = $language_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('config.factory'),
+      $container->get('language_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'language_negotiation_configure_url_suffix_form';
```

Comment:
Form IDs should be [namespaced with their module's name](https://www.drupal.org/docs/8/api/form-api/introduction-to-form-api#s-getformid).

Reply:
Corrected

Reply:
Maybe better to use `_settings` as it's the plugin settings form? So like
```
oe_multilingual_url_suffix_language_negotiation_settings
```